python_search/config.py

```diff
@@ -77,6 +79,8 @@ def __init__(
         default_tags=None,
         tags_dependent_inserter_marks: Optional[dict[str, Tuple[str, str]]] = None,
         default_text_editor: Optional[str] = None,
+        default_fzf_theme: Optional[str] = None,
+        custom_window_size: Optional[Tuple[int, int]] = None,
     ):
```

Comment:
Please also add to the documentation block below an explanation about this 2 variables following the standard of the others..